photon-core/src/main/java/org/photonvision/vision/camera/CameraQuirkConstants.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) Photon Vision.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.photonvision.vision.camera;
+
+/** Mr. Photonvision's Home for Peculiar Camera Constants */
+public class CameraQuirkConstants {
+    public static int[] LifecamAllowableExposures = {
+        5, 10, 20, 39, 78, 156, 312, 625, 1250, 2500, 5000, 10000, 20000
```

Comment:
how'd you find these numbers? document plz

Reply:
Copy-paste from https://github.com/wpilibsuite/allwpilib/blob/main/cscore/src/main/native/linux/UsbCameraImpl.cpp#L129

Probably worth a ping to peter to see what' he's got for history on those. I have a memory of a memory back in 2017 about finding something similar while trying to use one of these cameras from a beaglebone.